Signal/src/view controllers/MessageComposeTableViewController.m

```diff
@@ -156,9 +175,36 @@ - (void)createLoadingAndBackgroundViews {
     [_loadingBackgroundView addSubview:loadingProgressView];
     [_loadingBackgroundView addSubview:loadingLabel];
 
-    [self.noSignalContactsView.inviteButton addTarget:self
-                                               action:@selector(presentInviteFlow)
-                                     forControlEvents:UIControlEventTouchUpInside];
+    UIButton *inviteButton = self.noSignalContactsView.inviteButton;
```

Comment:
This is a bit of a hack - I'm leaving the Storyboard unmodified and making changes programmatically.  It doesn't seem that bad.